### PR TITLE
[ITS-365] - Formatting changes to 'Death of investor section'

### PR DESCRIPTION
**Pull request by sobrooks**
Updating the formatting on 'death of investor'

### CODE REVIEW

**Comment by victokoh** [comment]
@sobrooks can we get a screenshot of the page pls? before and after 🙏

File: source/includes/_isaadmin.md [hunk 1]
```diff
@@ -374,11 +374,13 @@ In the event of a death of an Investor, specific tasks need to be carried out. T
 ### 1) The investor's cash and/or investments are being transferred to a surviving spouse on the same platform using an Additional Permitted Subscription ('APS')
 
 If you want to transfer investments to a surviving spouse, you must make sure you have the appropriate legal paperwork and technical capability in place to transfer investments on your platform.  Alternatively, you can wait for investments to mature and only transfer cash when it becomes available.
+
 The process flow is:
+
 - Notify the Goji operations team of the death by emailing platformsupport@goji.investments.
-- Goji will create a 'Death of investor' workflow and lock the account.
+- Goji will create a 'Death of investor' workflow and lock the account.
```
Comment (victokoh) [inline source/includes/_isaadmin.md:381]: oh, is it just a space at the end here and removal of a space between sentences below?

Comment (victokoh) [reply source/includes/_isaadmin.md:381]: if that's it then, yeah, sure. lgtm

**Review by victokoh — APPROVED** [review]

File: source/includes/_isaadmin.md (hunk 1 above)
Comment (sobrooks) [reply source/includes/_isaadmin.md:381]: Yeah, that's it just a cleanup. Thanks!

**Pull request merged**